ddtrace/ext/app_types.go

```diff
@@ -75,4 +75,7 @@ const (
 
 	// SpanTypeConsul marks a span as a Consul operation.
 	SpanTypeConsul = "consul"
+
+	// SpanTypeGQLGen marks a span as a gqlgen operation.
+	SpanTypeGQLGen = "gqlgen"
```

Comment:
The value you've given is not a span type, it's the name of the repository. I realise this tag can be a bit confusing and is not very well defined. In practice, this represents the technology that the span is addressing and in our system (AFAIK) is only used in two places:
* obfuscation in the agent: some span types get obfuscated to cover for sensitive data (e.g. `sql`, `elasticsearch`, `redis`, etc.)
* in the app: some types get a special icon in a tab at the top of the trace view

The correct name and value for this would be `SpanTypeGraphQL = "graphql"`

Reply:
That makes sense. I haven't been clear on what exactly the span type is in the past, but this makes more sense.
The JS integration indeed uses "graphql" as the span type:
https://github.com/DataDog/dd-trace-js/blob/0673f03fc59abbad95810a006345bab2eca0ce8e/packages/datadog-plugin-graphql/src/index.js#L274-L277